src/services/public_http_server/handlers/notify_v1.rs

```diff
@@ -64,6 +64,18 @@ pub async fn handler_impl(
 ) -> Result<Response> {
     let start = Instant::now();
 
+    for notification in &body {
+        notification.notification.validate()?;
+    }
+
+    let subscriber_notification_count = body.iter().map(|n| n.accounts.len()).sum::<usize>();
+    const SUBSCRIBER_NOTIFICATION_COUNT_LIMIT: usize = 500;
+    if subscriber_notification_count > SUBSCRIBER_NOTIFICATION_COUNT_LIMIT {
+        return Err(Error::BadRequest(
```

Comment:
I would consider creating and throwing a `RateLimited` error for such cases.

Reply:
This isn't a rate limit, it's just a limit

Reply:
But, it's a sending rate limit for the client, so it's not like a malformed bad request which can be confusing. Anyway,  it's not blocking, just a suggestion.

Reply:
If anything it should be a 422 error instead of a 400. It's not a rate limit because they can call this up to the rate limit of the endpoint which is a separate number (and not implemented yet). It's a max size of the things you can do in 1 call, not the number of things you can do in a time period.